spec/controllers/auth/registrations_controller_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+describe Auth::RegistrationsController do
+
+  let(:valid_session) { {} }
+
+  describe 'POST #create' do
+
+    before :each do
+      request.env['devise.mapping'] = Devise.mappings[:user]
+    end
+
+    it 'defaults admin to false when omitted' do
+      post :create, user: {
+        username: 'portus',
+        email: 'portus@test.com',
+        password: '12341234',
+        'password_confirmation': '12341234'
+      }
+      assert !User.find_by!(username: 'portus').admin
```

Comment:
I would prefer to have written using rspec (like the rest of the tests).

Something like:
```ruby
expect(User.find_by!(username: 'portus').not_to be_admin
```